scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/Job.java

```diff
@@ -45,6 +45,7 @@
 import java.net.MalformedURLException;
 import java.util.HashMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.HashMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)